Library/Homebrew/cmd/--cache.rb

```diff
@@ -32,6 +32,8 @@ def __cache_args
              description: "Only show cache files for casks."
       conflicts "--build-from-source", "--force-bottle"
       conflicts "--formula", "--cask"
+
+      named_args [:formula, :cask]
```

Comment:
Maybe this could/should be `:formula_or_cask` because we don't want to allow e.g. `[:formula, :tap]` ever?

Reply:
We do have one instance (kind of). In `style` we use `[:formula, :cask, :tap]`.

We also have `bump-unversioned-casks` that is `[:cask, :tap]`, so I think it's possible that we would want a `[:formula, :tap]` in the future. I'm not sure we want to forbid it.

My preference is to use the array because it says "a named argument can be of any of these types". If we add `:formula_or_cask`, it makes things a tad more complicated.

Reply:
> We do have one instance (kind of). In `style` we use `[:formula, :cask, :tap]`.

Gotcha, this is fine as-is IMO then 👍🏻